shared/constants/teams.js

```diff
@@ -131,6 +137,7 @@ export const makeState: I.RecordFactory<_State> = I.Record({
   teamNameToMemberUsernames: I.Map(),
   teamNameToMembers: I.Map(),
   teamNameToRequests: I.Map(),
+  teamNameToIsOpen: I.Map(),
```

Comment:
Maybe this should be `teamNameToSettings`, since we're adding the showcasing setting and potentially more in the future.